.github/workflows/close-stale-issues.yml

```diff
@@ -0,0 +1,36 @@
+name: Close stale issues
+on:
+  workflow_dispatch:
+  schedule:
+    # Run every day at 1:00 AM UTC.
+    - cron: 0 1 * * *
+
+# yamllint disable rule:empty-lines
+env:
+  CLOSE_MESSAGE: >
+    There has been no activity on this issue
+    and it is being closed. If you feel closing this issue is not the
+    right thing to do, please leave a comment.
+  WARN_MESSAGE: >
+    There has been no activity on this issue for
+    3 years and it may no longer be relevant.
+    It will be closed 1 month after the last non-automated comment.
```

Comment:
I thought about adding info about where better places to post help might be, but per conversation within https://github.com/nodejs/admin/issues/757 I think we can proactively put that on the front-end of issue creation (template, README, https://github.com/marketplace/actions/welcome-new-users)

these messages over in https://github.com/nodejs/node/pull/42085 also included a link to docs on how feature requests are managed. If we need to document this policy somewhere, I am happy to add that too (I just don't know where, or if it's even needed).